components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/dao/impl/ApplicationDAOImpl.java

```diff
@@ -1523,28 +1570,67 @@ private int getApplicationIDByName(String applicationName, int tenantID, Connect
         return applicationId;
     }
 
+
+    /**
+     * Reading the mapping of properties.
+     *
+     * @param customAuthenticator
+     * @param propertyName
+     * @return
+     */
+    private Property getMappedProperty(AbstractInboundAuthenticatorConfig customAuthenticator, String propertyName) {
+        Property property = null;
```

Comment:
if customAuthenticator or propertyName is null return at the beginning of the method